invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""MARCXML based Schema for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import bleach
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing
+
+from ..ui.schema import current_default_locale
+from ..utils import get_vocabulary_props
+
+
+class MARCXMLSchema(Schema):
+    """Schema for records in MARC."""
+
+    contributors = fields.Method("get_contributors")
+    titles = fields.Method("get_titles")
+    creators = fields.Method("get_creators")
+    identifiers = fields.Method("get_identifiers")
+    relations = fields.Method("get_relations")
+    rights = fields.Method("get_rights")
+    dates = fields.Method("get_dates")
+    subjects = fields.Method("get_subjects")
+    descriptions = fields.Method("get_descriptions")
+    publishers = fields.Method("get_publishers")
+    types = fields.Method("get_types")
+    sources = fields.Method("get_sources")
+    coverage = fields.Method("get_locations")
+    formats = fields.Method("get_formats")
+
+    def get_titles(self, obj):
+        """Get titles."""
+        return [obj["metadata"]["title"]]
+
+    def get_identifiers(self, obj):
+        """Get identifiers."""
+        items = []
+        items.extend(i["identifier"] for i in obj["metadata"].get("identifiers", []))
+        items.extend(p["identifier"] for p in obj.get("pids", {}).values())
+
+        return items or missing
+
+    def get_creators(self, obj):
+        """Get creators."""
+        return [c["person_or_org"]["name"] for c in obj["metadata"].get("creators", [])]
+
+    def get_relations(self, obj):
+        """Get relations."""
+        rels = []
+
+        # Fundings
+        # FIXME: Add after UI support is there
```

Comment:
leftover? why is not possible now?

Reply:
I assumed this was not possible as it isn't implemented in
https://github.com/TLGINO/invenio-rdm-records/blob/d1052925f1106be65c80d4c94d91d3b31572837d/invenio_rdm_records/resources/serializers/dublincore/schema.py#L19
If it is, then I can change it in both implementations